karapace/schema_reader.py

```diff
@@ -248,56 +249,59 @@ def handle_messages(self) -> None:
                 try:
                     key = ujson.loads(msg.key.decode("utf8"))
                 except ValueError:
-                    self.log.exception("Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
+                    self._log.exception("Invalid JSON in msg.key")
+                    self._log.log(TRACE, "Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
                     continue
 
                 value = None
                 if msg.value:
                     try:
                         value = ujson.loads(msg.value.decode("utf8"))
                     except ValueError:
-                        self.log.exception("Invalid JSON in msg.value: %r, key: %r", msg.value, msg.key)
+                        self._log.exception("Invalid JSON in msg.value")
+                        self._log.log(TRACE, "Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
                         continue
 
-                self.log.info("Read new record: key: %r, value: %r, offset: %r", key, value, msg.offset)
-                self.handle_msg(key, value)
-                self.offset = msg.offset
-                self.log.info(
-                    "Handled message, current offset: %r, ready: %r, add_offsets: %r",
-                    self.offset,
-                    self.ready,
+                self._log.log(
+                    TRACE,
+                    "New record: key: %r, value: %r, offset: %r, add_offsets: %r",
+                    key,
+                    value,
+                    msg.offset,
                     add_offsets,
                 )
+                self.handle_msg(key, value)
+                self.offset = msg.offset
                 if self.ready and add_offsets:
                     self.offset_watcher.offset_seen(self.offset)
 
     def _handle_msg_config(self, key: dict, value: Optional[dict]) -> None:
         subject = key.get("subject")
         if subject is not None:
             if subject not in self.subjects:
-                self.log.info("Adding first version of subject: %r with no schemas", subject)
+                self._log.info("Adding first version of subject: %r with no schemas", subject)
```

Comment:
I thought about not logging `subject`s too, but I think in that case we would have too little logging available.